### PR TITLE
Add microfrontends to distro

### PR DESCRIPTION
**Pull request by brandones**

### CODE REVIEW

**Review by mseaton — APPROVED** [review]
Looks good to me @brandones !  Happy to iterate once this is in place and we can see how it's working.

File: distro/microfrontends.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "microfrontends": {
+    "@pih/esm-refapp-navbar-app": "latest",
+    "@pih/esm-referrals-queue-app": "latest",
+    "@openmrs/esm-home-app": "next",
```
Comment (mogoodrich) [inline distro/microfrontends.json:5]: Out of curiosity, what does "next" mean in terms of version?

**Review by mogoodrich — COMMENTED** [review]
Belated LGTM!

File: distro/microfrontends.json (hunk 1 above)
Comment (brandones) [reply distro/microfrontends.json:5]: It's a tag. Generally (and in our case) it's used for the latest non-release version. So e.g. `latest` might be 1.1.0 but `next` might be `1.2.0-beta3`.

Comment (mogoodrich) [reply distro/microfrontends.json:5]: Ah, so it's basically the equivalent of "snapshot" from maven... sounds good. And probably a better word that "snapshot"... I hated "snapshot" at first but got used to it.

**Pull request merged**